wikimed/info.json

```diff
@@ -0,0 +1,13 @@
+{
+    "about_app_url": "https://www.kiwix.org/support",
+    "about_text": "Offline Medical Wikipedia",
```

Comment:
I wasn't sure about this part, but now it's easy to edit it.

Reply:
Ah right we don't have this on android.